run_benchmarks.rb

```diff
@@ -432,6 +438,9 @@ def run_benchmarks(ruby:, ruby_description:, categories:, name_filters:, out_pat
   if have_yjit?(RbConfig.ruby)
     args.executables["interp"] = [RbConfig.ruby]
     args.executables["yjit"] = [RbConfig.ruby, "--yjit", *args.yjit_opts.shellsplit]
+  elsif have_zjit?(RbConfig.ruby)
```

Comment:
Do we want zjit checked first, or yjit first?
Are we generally building rubies with both, or just one of them?